target/linux/ath79/generic/config-default

```diff
@@ -14,6 +14,7 @@ CONFIG_LEDS_RESET=y
 CONFIG_MARVELL_PHY=y
 CONFIG_MICREL_PHY=y
 CONFIG_MTD_REDBOOT_PARTS=y
+CONFIG_MTD_SPI_NOR_USE_VARIABLE_ERASE=y
```

Comment:
Is `CONFIG_MTD_SPI_NOR_USE_VARIABLE_ERASE` really needed for this device? 
I think changing it will cause problem with other devices.

Reply:
It is required because the dpt-persistent partition must be writable from within OpenWRT. Therefore a block size of 32kB instead of 64kB is required. If it can be set device specific this would also solve it